README.md

```diff
@@ -31,7 +31,8 @@ To install the Scorecards GitHub Action, you need to:
 
 ### Authentication
 1. [Create a Personal Access Token](https://github.com/settings/tokens/new) with the following read permissions:
-    - Note: `Read-only token for OSSF Scorecard Action - myorg/myrepo` (Note: replace `myorg/myrepo` with the names of your organization and repository so you can keep track of your tokens.)
+    - Note 1: `Read-only token for OSSF Scorecard Action - myorg/myrepo` (Note: replace `myorg/myrepo` with the names of your organization and repository so you can keep track of your tokens.)
```

Comment:
Not sure I understand the point being made in `Note 1`.

Reply:
I think that the idea was that `Note` (in the original) was one of the fields that needed to be filled in while creating the token. Then it got confusing because I added "Note: replace `myorg/myrepo` with...). So we can delete that usage of "note" in the paranthses and just say "Replace `myorg/myrepo`....)

This new addition of Note 1 and Note 2 makes it even more confusing. I'd suggest taking out 1 and 2 and leaving the original like this:
 "Note: `Read-only token for OSSF Scorecard Action -- myorg/my repo` (Replace `myorg/my repo` with the of your organization and repository so you can keep track of your tokens)."

Then remove Note 2 and move that point elsewhere.  Hope that all makes sense!

Reply:
yes, I just realized that! ptal as I've made changes